rotkehlchen/tests/api/test_addressbook.py

```diff
@@ -1020,3 +1021,27 @@ def test_insert_unsupported_ecosystem_address_with_none_blockchain(
         contained_in_msg='Given address is from an unsupported ecosystem',
         status_code=HTTPStatus.BAD_REQUEST,
     )
+
+
+@pytest.mark.parametrize('empty_global_addressbook', [True])
+@pytest.mark.parametrize('book_type', [AddressbookType.PRIVATE])
+def test_insert_solana_address(
```

Comment:
do we have a test for non-solana, non-btc, non-evm addresses? AKA random strings?